kava-stats.js

```diff
@@ -391,19 +394,9 @@ const mapCdpParams = async (denoms, cdpParamsData) => {
       if(cdpParam) { limit = cdpParam.debtLimit }
     }
 
-    coins[denom] = { debtLimit: limit }
+    coins[denom] = { debtLimit: limit, stabilityFeePercentage: mappedStabilityFees[denom] }
   }
 
-      for (const denom of cdpParamsData.collateral_params) {
```

Comment:
instead of looping through cdp params again here, we can reuse the cdp params loop above.  Looks like you may have already started this approach by adding stability fee at line 379.